src/components/room-bridge-store.ts

```diff
@@ -274,7 +274,7 @@ export class RoomBridgeStore extends BridgeStore {
                 return e.remote;
             }) as RemoteRoom[];
         }
-        return entryMap;
+        return result;
```

Comment:
:D Nice catch.
(For how long has this been broken? D: )

Reply:
Only for the Typescript bits, luckily.